region/maridia/inner-yellow/Watering Hole.json

```diff
@@ -224,6 +224,98 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "G-Mode Morph - Remote Acquire Left Item",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canOffScreenMovement",
+        {"or": [
+          {"and": [
+            "canTrickyJump",
+            "h_canArtificialMorphBombs",
+            "h_canArtificialMorphSpringBall"
+          ]},
+          {"and": [
+            "canInsaneJump",
+            "h_canArtificialMorphBombs"
+          ]},
+          {"and": [
+            "h_canArtificialMorphPowerBomb",
```

Comment:
Can't tell if its intended, but PBs + DoubleSpringball jump does cleanly kill the enemies and lets you move through the room.